src/DataStructures/DenseVector.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class DenseVector.
+
+#pragma once
+
+#include <pup.h>  // IWYU pragma: keep
+
+#include "Utilities/Blaze.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+#include <blaze/math/DynamicVector.h>
+#include <blaze/system/Version.h>
+
+/*!
+ * \ingroup DataStructuresGroup
+ * \brief A dynamically sized vector of arbitrary type.
+ *
+ * \details Use this vector type to represent a generic collection of numbers
+ * with no particular meaning. It supports all common vector operations such as
```

Comment:
Here again the phrase "no particular meaning", to me, is saying "does not have _any_ particular meaning". I suggest something like
> Use this vector type to represent a generic collection of numbers without assigning any particular meaning to them.